server/schemas/schemaController.js

```diff
@@ -44,6 +44,7 @@ var handleUpload = function(request, response, appendData) {
             } else {
               console.log("successfully created", filename);
             }
+            helpers.endFormParse(response);
```

Comment:
Originally our success response could happen before the file had actually uploaded. The sending of the response has been moved to the callback that executes when the upload is complete.